addons/mail/static/src/core/common/attachment_list.xml

```diff
@@ -9,90 +9,88 @@
-                        <div class="overflow-auto d-flex justify-content-center flex-column px-1">
-                            <div t-if="attachment.name" class="text-truncate" t-out="props.messageSearch?.highlight(attachment.name) ?? attachment.name"/>
-                            <small t-if="attachment.extension" class="text-uppercase" t-esc="attachment.extension"/>
-                        </div>
-                    </t>
-                    <div class="o-mail-AttachmentCard-aside position-relative rounded-end overflow-hidden" t-att-class="{ 'o-hasMultipleActions d-flex flex-column': attachment.isDeletable and !env.inComposer }">
-                        <div t-if="attachment.uploading" class="d-flex justify-content-center align-items-center w-100 h-100" title="Uploading">
+                <t t-foreach="this.props.attachments" t-as="attachment" t-key="attachment.id">
+                    <div t-if="attachment.isImage" t-att-aria-label="attachment.filename"
+                        class="o-mail-AttachmentImage d-flex position-relative flex-shrink-0 mw-100 mb-1 me-1"
+                        t-att-title="attachment.name"
+                        t-att-class="{ 'o-isUploading': attachment.uploading }"
+                        tabindex="0"
+                        aria-label="View image"
+                        t-att-data-mimetype="attachment.mimetype"
+                        t-on-click="() => this.fileViewer.open(attachment, props.attachments)"
+                        role="menu"
+                    >
+                        <img
+                            class="img img-fluid my-0 mx-auto o-viewable"
+                            t-att-class="{ 'opacity-25': attachment.uploading }"
+                            t-att-src="getImageUrl(attachment)"
+                            t-att-alt="attachment.name"
+                            t-attf-style="max-width: min(100%, {{ imagesWidth }}px); max-height: {{ props.imagesHeight }}px;"
+                            t-on-load="onImageLoaded"
+                        />
+                        <div t-if="attachment.uploading" class="position-absolute top-0 bottom-0 start-0 end-0 d-flex align-items-center justify-content-center" title="Uploading">
                             <i class="fa fa-spin fa-spinner"/>
                         </div>
-                        <div t-if="!attachment.uploading and env.inComposer" class="d-flex justify-content-center align-items-center w-100 h-100 text-primary" title="Uploaded">
-                            <i class="fa fa-check"/>
+                        <div class="position-absolute top-0 bottom-0 start-0 end-0 p-2 text-white opacity-0 opacity-100-hover d-flex align-items-end flax-wrap flex-column">
+                            <div t-if="showDelete"
```

Comment:
issue at rebase? condition on `!ui.isSmall` seems to have been removed

Reply:
Hello, the comment was marked as resolved but the condition is still removed, could you explain the reasoning?

Reply:
I added them on the buttons for testing purposes, but I overlooked them and failed to replace them back to their original position.